crates/holochain/src/sweettest/sweet_agents.rs

```diff
@@ -36,6 +37,11 @@ impl SweetAgents {
         (agents.pop().unwrap(), agents.pop().unwrap())
     }
 
+    /// Get the same two AgentPubKeys every time
+    pub fn alice_and_bob() -> (AgentPubKey, AgentPubKey) {
```

Comment:
This is incorrect. These agents need to come from the keystore as above. Any reason to not just use `two()`? Why do the agent keys need to be the same every time?

Reply:
the agent keys are the same every time so we can test our crypto deterministically, these agent's private keys are in the keystore already it's just that we need their public keys exposed so we can reference them